src/pages/RegisterStaff.jsx

```diff
@@ -0,0 +1,82 @@
+import { RegisterStaffForm } from '@/forms/staff/forms';
+import { registerStaffSchema } from '@/forms/staff/schemas';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card';
+import { registerStaff } from '@/services/staff';
+import { useState } from 'react';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router';
+
+export function RegisterStaff() {
+    
+  return (
+    <div className="flex min-h-screen items-center justify-center">
+      {/* Right side - Register Staff Form */}
+      <div className="w-full lg:w-2/3 flex items-center justify-center p-8">
+        <RegisterStaffCard />
+      </div>
+    </div>
+  );
+}
+
+function RegisterStaffCard() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState('');
+  const navigate = useNavigate();
+
+  const form = useForm({
+    resolver: zodResolver(registerStaffSchema),
+    defaultValues: {
+      email: '',
+      password: '',
+      name: '',
+      surname: '',
+      specialty: '',
+      dni: '',
+    },
+  });
+
+  function onSubmit(values) {
+    setIsLoading(true);
+    registerStaff(values)
+      .then((response) => {
+
+        if (response.status === 201)
+          navigate('/app');
```

Comment:
Here maybe just return to previous page with
```javascript
navigate(-1)
```

Reply:
When a page to visualize a doctor profile is ready, I will use that to navigate to